iosApp/iosApp/ComponentViews/ErrorBanner.swift

```diff
@@ -46,6 +46,12 @@ struct ErrorBanner: View {
                 }
                 .frame(minHeight: minHeight)
             }
+        case .networkError:
+            ErrorCard { HStack {
+                Image(systemName: "wifi.slash")
+                Text("Unable to connect")
```

Comment:
Because this has no reload button like the other two banners, it also has no `Spacer()`, so we're getting the awkward squished banner. If you toss a `Spacer()` at the end here, I think that'll get it to take up the full width.